### DISCUSSION TITLE
Container: Difference between maxW and maxWidth?

### DISCUSSION THREAD

**Discussion by chug2k** [chakra-ui/chakra-ui#3626] [Q&A]
Hi there,

Getting used to Chakra, and I'm trying to make a container. I'm seeing on the documentation page that you can set either maxW or maxWidth, and I'm confused about the difference on each prop. 

As a follow up - I'd like all containers by default to be 80ch wide, not 60ch wide, and having a hard time how to fix my theme to set this. My guess was to set a maxWidth under `defaultProps` for my Container, but that doesn't seem to be working.

**Comment by malashevskyi** [comment]
Hi. I think 'maxWidth' the same as 'maxW' I didn't see maxWidth usage in docs, where did you see that? Could you give a link?

I don't know is it possible to change container size in theme,  but another ease way to do that is:
```javascript
<Container maxW="80ch" />
```

**Reply by chug2k** [reply]
https://chakra-ui.com/docs/layout/container, near the top: 

> By default, it sets the max-width of the content to 60 characters (60ch) but you can customize this by passing custom maxWidth values or changing the container size tokens defined in theme.sizes.container.

Thanks so much for the reply and follow-up. I think that threw me off, it looks like that should be `maxW` instead of `maxWidth`?

**Reply by malashevskyi** [reply]
`maxW` and `maxWidth` work the same (at least for me), but if you set `maxWidth` for `sizes` or `baseStyle` you will be unable to override it with `maxW` on component (`<Container maxW={} />` - will not work)
I use only maxW everywhere

**Comment by malashevskyi** [comment]
**"As a follow up - I'd like all containers by default to be 80ch wide, not 60ch wide"**
Found a solution:
```javascript
const Container = {
  baseStyle: {
    maxW: '80ch'
  },
}

const theme = extendTheme({
  components: {
    Container
  },
})
```

**Reply by chug2k** [reply]
Ah, this works. Thank you! In the docs (which is the same thing I copied above):

>  changing the container size tokens defined in `theme.sizes.container`.

I wasn't sure how to do this one. Is this the "more correct" way to do it?

**Reply by malashevskyi** [reply]
I don't think that there is more correct or less, you can use `baseStyle` or sizes:
```javascript
const Container = {
  sizes: {
    myCustomSize: {
      maxW: '80ch',
      // other properties
    }
  },

  defaultProps: {
    size: 'myCustomSize'
  }
}

const theme = extendTheme({
  components: {
    Container
  },
})
```
